framework/OpenMod.Runtime/Runtime.cs

```diff
@@ -291,6 +295,35 @@ public async Task PerformSoftReloadAsync()
             await InitAsync(m_OpenModHostAssemblies!, m_RuntimeInitParameters!, m_HostBuilderFunc);
         }
 
+        private void PerformFileSystemWatcherPatch()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && Type.GetType("Mono.Runtime") != null)
+            {
+                var watcherField = typeof(FileSystemWatcher).GetField("watcher", BindingFlags.Static | BindingFlags.NonPublic);
+                var watcher = watcherField?.GetValue(null);
+
+                if (watcher?.GetType().GetField("watches", BindingFlags.Static | BindingFlags.NonPublic)?.GetValue(watcher) is Hashtable watches)
+                {
+                    FieldInfo? incSubdirsField = null;
+
+                    // DictionaryEntry<FileSystemWatcher, DefaultWatcherData>
+                    foreach (var data in watches)
+                    {
+                        if (data is DictionaryEntry entry)
+                        {
+                            if (entry.Key is FileSystemWatcher fileSystemWatcher)
+                            {
+                                fileSystemWatcher.IncludeSubdirectories = false;
```

Comment:
Could be improved to use fileSystemWatcher._watcher.IncludeSubdirectories instead of disabling it for everyone